.github/workflows/kakarot_rpc.yml

```diff
@@ -118,3 +119,29 @@ jobs:
             --tag ${{ env.REGISTRY_IMAGE }}:${{ github.ref_name }} \
             --tag ${{ env.REGISTRY_IMAGE }}:latest \
             --push
+
+  build-push-indexer-docker:
+    runs-on: ubuntu-latest-16-cores
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ env.DOCKER_USERNAME }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          push: true
+          tags: ${{ env.INDEXER_IMAGE }}:${{ github.ref_name }}, ${{
+            env.INDEXER_IMAGE }}:latest
+          context: .
+          file: ./docker/indexer/Dockerfile
+          platforms: linux/amd64
```

Comment:
only building for amd, maybe should add arm 🤷

Reply:
might need to check with Karnot

Reply:
We will be needing ARM support because most of the container runs on ARM64 arch nodes